podcasts/Categories/CategoriesSelectorViewController.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import PocketCastsServer
+
+class CategoriesSelectorViewController: UIHostingController<CategoriesSelectorView>, DiscoverSummaryProtocol {
+
+    class Observable: ObservableObject {
```

Comment:
> Can we rename that to DiscoverItemObservable?

Sure! I was thinking it more as the observable object tied to the view controller so it could have other things than the Discover Item but we can change it if that ever happens.